### ISSUE TITLE
Having issues with three of the files- "expects" "higher order functions" and "inheritance" are not showing up when I refresh the browser. Those subjects have disappeared!

### ISSUE THREAD

**Issue by ghost** [mrdavidlaing/javascript-koans#38]
In summary. there were 8 files and subjects. when I refresh the browser, it only shows 5 files and subjects.  "expects" "higher order functions" and "inheritance" are not showing up when I refresh the browser. Those subjects have disappeared!

Here's what it looks like:

![image](https://cloud.githubusercontent.com/assets/9400036/5208639/2233429c-7584-11e4-8842-b4694c104371.png)

**Comment by j0narch3r** [comment]
Ditto, same problem, though different subjects. Arrays and Objects also disappeared for me before finishing them (somewhere around the 3rd or 4th test).

**Comment by wnmurphy** [comment]
I was experiencing the same problem throughout: I'd get something incorrect, one of the subjects disappeared from the "Subjects covered" count at the top of KoansRunner.html, the "Koans Learned" count decreased, and KoansRunner.html jumped to the next subject prematurely.

I found that this occurred whenever I had a typo, for example if I closed an array with two right brackets. You could try pasting the problematic section of your code into http://jshint.com to see what's wrong; just ignore the undefined variable errors.

[Same comment posted on issue #45 Disappearing Koan... same fix for same issue.]

**Comment by v-node** [comment]
@j0narch3r do you mean you started the koans from the beginning?

**Comment by j0narch3r** [comment]
@v-node Yes. The only way I could see to guarantee I was getting through all of them and not missing subjects to the bug.

**Comment by krispyokreme** [comment]
I have the same issue happening with most of the koans. I always refreshed through Cmd + R and never clicked on the "Meditate again" links. I've checked over my code many times to make sure there were no typos (such as an extra closing bracket). I've even redownloaded the files again and started over, but still had the same issue. Does anyone know what's causing this? Could it be the browser? I'm using Firefox version 37.0.2. Do I need to upgrade to a newer version? Does this issue occur for people using Chrome? Thanks.

**Comment by freezycold84** [comment]
Would be great if someone fixed this



Thanks,
Fareez

**Comment by aboutabel** [comment]
I had this issue, until I fixed a couple typos with an extra closing bracket in my answers. Using Chrome.

**Comment by Josamuel** [comment]
I had the same issue.  I actually got it when I added a toString function to a number.  Sure enough editing that out fixed it.
